HomeToGather/HomeToGather/View/TicketList/Invited/InvitedListView.swift

```diff
@@ -48,6 +49,13 @@ let data: [Invitation] = [
 
 struct InvitedListView: View {
     let screenWidth = UIScreen.main.bounds.width
+    let uid = Auth.auth().currentUser!.uid
+    
+    @State var viewModel: ViewModel = ViewModel()
```

Comment:
```suggestion
    @State var viewModel = ViewModel()
```

Swift에서 명확한 타입의 경우에는 타입표기를 생략할 수 있답니다 : )

Reply:
그럼 혹시 코드를 작성할 때 명확한 타입 선언은 지양하는 편이 좋을까요?

Reply:
Int, UInt, Double, CGFloat, Float처럼 다양한 타입이 될 수 있는 숫자가 아닌, 명확한 경우에는 Swift를 믿고 생략해도 괜찮아요 !